src/IndirectArrays.jl

```diff
@@ -40,6 +40,16 @@ Base.IndexStyle(::Type{IndirectArray{T,N,A,V}}) where {T,N,A,V} = IndexStyle(A)
 
 Base.copy(A::IndirectArray) = IndirectArray(copy(A.index), copy(A.values))
 
+# This method is only necessary because of a bug in Julia 0.6.2 and can be removed
```

Comment:
Put this under a `VERSION` check (with the commit know to fix it)? That way Femtocleaner will remove it automatically.

Reply:
Good idea. I've done that.